pkg/network/bridge/lister.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	resourceNamespace              = "bridge.network.kubevirt.io"
+	ResourceNamespace              = "bridge.network.kubevirt.io"
```

Comment:
i think you can keep it private. the code using it belongs to the same package (even though it is in a different file).

Reply:
Thanks, I was not aware of this.